### PR TITLE
Feature/state machine

### PR DESCRIPTION
**Pull request by vdv73rus**

### CODE REVIEW

File: test/factories/tasks.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :task do
+    name
+    description
+    author { create :manager }
```
Comment (artempartos) [inline test/factories/tasks.rb]: it's better to have nil in assoc by default, and create trait like (with_author) if you need task with author

**Comment by artempartos** [comment]
PR -> develop

File: test/factories/tasks.rb (hunk 1 above)
Comment (vdv73rus) [reply test/factories/tasks.rb]: Ok, I'll remove it for now. BTW sequences and factories are in the next chapter 'Registration & Enter', the lesson 'Working with tests'

**Comment by artempartos** [comment]
👍

**Pull request merged**